cni/network/invoker_cns.go

```diff
@@ -151,18 +151,35 @@ func setHostOptions(nwCfg *cni.NetworkConfig, hostSubnetPrefix *net.IPNet, ncSub
 		},
 	}
 
-	azureDNSMatch := fmt.Sprintf(" -m addrtype ! --dst-type local -s %s -d %s -p %s --dport %d", ncSubnetPrefix.String(), iptables.AzureDNS, iptables.UDP, iptables.DNSPort)
-
+	ncSubnetPrefixStr := ncSubnetPrefix.String()
+	azureDNSMatch := fmt.Sprintf(" -m addrtype ! --dst-type local -s %s -d %s -p %s --dport %d", ncSubnetPrefixStr, iptables.AzureDNS, iptables.UDP, iptables.DNSPort)
```

Comment:
Just curious: is there a reason for us to specify TCP here ? my understanding is DNS can be UDP or TCP and we should be allowing any of the protos if dport is 53